### PR TITLE
wayback_machine_downloader: init at 2.2.1

### PR DESCRIPTION
**Pull request by manveru**
###### Motivation for this change

Just used it myself

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/wayback_machine_downloader/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{ lib, bundlerApp, bundlerUpdateScript }:
+bundlerApp {
+  pname = "wayback_machine_downloader";
+  exes = [ "wayback_machine_downloader" ];
+  gemdir = ./.;
+
+  passthru.updateScript = bundlerUpdateScript "wayback_machine_downloader";
+
+  meta = with lib; {
+    description =
```
Comment (alexarice) [inline pkgs/applications/networking/wayback_machine_downloader/default.nix]: `description` should not end with a full stop

File: pkgs/applications/networking/wayback_machine_downloader/default.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "Download an entire website from the Internet Archive Wayback Machine.";
```
Comment (mmahut) [inline pkgs/applications/networking/wayback_machine_downloader/default.nix]: Please keep it on one line.

**Comment by mmahut** [comment]
@GrahamcOfBorg build wayback_machine_downloader

**Comment by mmahut** [comment]
@GrahamcOfBorg build wayback_machine_downloader

**Pull request merged**